cranelift/codegen/src/isa/s390x/lower/isle.rs

```diff
@@ -471,4 +503,9 @@ where
     fn emit(&mut self, inst: &MInst) -> Unit {
         self.emitted_insts.push((inst.clone(), false));
     }
+
+    #[inline]
+    fn emit_safepoint(&mut self, inst: &MInst) -> Unit {
+        self.emitted_insts.push((inst.clone(), true));
+    }
```

Comment:
And then I would expect this to go into `wasmtime/cranelift/codegen/src/machinst/isle.rs`